tests/robustness/validate/patch_history.go

```diff
@@ -214,6 +214,9 @@ func putReturnTime(allOperations []porcupine.Operation, reports []report.ClientR
 					continue
 				}
 				kv := keyValue{Key: op.Put.Key, Value: op.Put.Value}
+				if count := persistedPutCount[kv]; count > 1 {
```

Comment:
I think we should rather test uniqueness of `clientPutCount` instead of `persistedPutCount`, it should be more obvious for reader, while there is not much difference in effect.